cpp/src/Slice/MetadataValidation.cpp

```diff
@@ -95,26 +94,27 @@ Slice::validateMetadata(const UnitPtr& p, string_view prefix, map<string, Metada
          typeid(DataMember)},
         MetadataArgumentKind::OptionalTextArgument,
     };
+    knownMetadata.emplace("deprecate", std::move(deprecatedInfo)); // Kept as an alias for 'deprecated'.
```

Comment:
I forgot to add this in my original metadata validation PR.
We don't actually use it in our Slice files, but we decided to keep support for the `3.7` name for backwards compatibility. We don't want to bother our users about adding a single `d` character to their metadata.